src/test/java/unstable/MarkAllIssuesAsReadUnreadTests.java

```diff
@@ -0,0 +1,107 @@
+package unstable;
+
+import guitests.UITest;
+import javafx.application.Platform;
+import javafx.scene.control.ContextMenu;
+import javafx.scene.control.MenuItem;
+import javafx.scene.input.KeyCode;
+import org.junit.Before;
+import org.junit.Test;
+import org.loadui.testfx.GuiTest;
+import ui.components.FilterTextField;
+import ui.components.IssueListView;
+import ui.listpanel.ListPanel;
+import ui.listpanel.ListPanelCell;
+import util.PlatformEx;
+
+import java.lang.reflect.*;
+
+import static org.junit.Assert.assertEquals;
+
+public class MarkAllIssuesAsReadUnreadTests extends UITest {
+
+    private static final String PANEL_IDENTIFIER = "#dummy/dummy_col0";
+
+    @Before
+    public void setup() {
+        PlatformEx.runAndWait(stage::requestFocus);
+
+        FilterTextField filterTextField = find(PANEL_IDENTIFIER + "_filterTextField");
+        filterTextField.setText("");
+
+        click(PANEL_IDENTIFIER + "_filterTextField");
+        push(KeyCode.ENTER);
+    }
+
+    /**
+     * Tests selecting "Mark all below as read" and "Mark all below as unread" context menu items
+     */
+    @Test
+    public void markAllBelowAsReadUnread_multipleIssuesInListView_issuesCorrectlyMarked()
+            throws NoSuchFieldException, IllegalAccessException {
+
+        ListPanel issuePanel = find(PANEL_IDENTIFIER);
+        Field listViewField = ListPanel.class.getDeclaredField("listView");
+        listViewField.setAccessible(true);
+        IssueListView listViewValue = (IssueListView) listViewField.get(issuePanel);
+
+        // scrolls to the end of the panel
+        listViewValue.scrollAndShow(issuePanel.getIssueCount());
+
+
+        //checking for issue #7 and below as marked read
+        clickAndMarkIssuesBelow(issuePanel, 7, true);
+        verifyReadStatusOfIssuesBelow(issuePanel, 7, true);
+
+        //checking for issue #7 and below as marked read
```

Comment:
Mention that you are doing it again, and why you are doing it again. The comment is supposed to give the missing info, not repeat the obvious info.